sdk/core/core-paging/package.json

```
@@ -35,15 +35,22 @@
     "audit": "node ../../../common/scripts/rush-audit.js && rimraf node_modules package-lock.json && npm i --package-lock-only 2>&1 && npm audit",
     "build": "tsc -p .",
     "build:test": "tsc -p .",
-    "clean": "",
+    "check-format": "echo skipped",
+    "clean": "echo skipped",
     "format": "prettier --write --config .prettierrc.json \"src/**/*.ts\"",
-    "integration-test": "",
+    "integration-test:browser": "echo skipped",
+    "integration-test:node": "echo skipped",
+    "integration-test": "npm run integration-test:node && npm run integration-test:browser",
     "lint": "tslint -p .",
```

We have switched to eslint, we should use it here:
```suggestion
     "lint": "eslint -c ../../.eslintrc.json src test --ext .ts -f node_modules/eslint-detailed-reporter/lib/detailed.js -o template-lintReport.html || exit 0",
```